extensions/ql-vscode/src/stories/common/SuggestBox.stories.tsx

```diff
@@ -141,6 +142,25 @@ export const AccessPath = Template.bind({});
 AccessPath.args = {
   options: suggestedOptions,
   parseValueToTokens: (value: string) => value.split("."),
+  validateValue: (value: string) => {
+    let index = value.indexOf("|");
+
+    const diagnostics: Diagnostic[] = [];
+
+    while (index !== -1) {
+      index = value.indexOf("|", index + 1);
```

Comment:
Probably a silly question: Why is `|` specifically disallowed? Is it a particular Ruby thing, or more to do with the CodeQL/model editor in general? (i.e. would we have to disallow other symbols for other languages?)

Reply:
Sorry, this is just for testing so we don't rely on access path-specific code. This was just easy to implement. I'll add a comment to clarify that.